package.json

```diff
@@ -33,11 +33,13 @@
   "dependencies": {
     "canonical-json": "0.0.4",
     "csv-parse": "^4.4.3",
+    "dom-compare": "^0.6.0",
     "eslint": "^6.5.0",
     "eslint-loader": "^3.0.2",
     "googleapis": "^43.0.0",
     "iso-639-1": "^2.1.0",
-    "json2csv": "^4.5.1",
+    "json-diff": "^0.5.4",
+    "json2csv": "^4.3.5",
```

Comment:
This appears to have regressed - can we go back to the 4.5.1 version or was there an issue with it?

Reply:
Did not notice the change. Good catch! Fixed